fil-proofs-tooling/src/bin/gpu-cpu-test/main.rs

```diff
@@ -34,11 +34,24 @@ const POST_CONFIG: PoStConfig = PoStConfig {
     api_version: FIXED_API_VERSION,
 };
 
-arg_enum! {
-    #[derive(Debug)]
-    pub enum Mode {
-        Threads,
-        Processes,
+#[derive(Debug, Clone)]
+pub enum Mode {
+    Threads,
+    Processes,
+}
+
+impl FromStr for Mode {
```

Comment:
Technically, one can derive `ArgEnum`. But that doesn't actually work with `type_of_t`, so I'm not sure what one is supposed to do here. But this version _should_ actually work.